modules/distribution/src/repository/resources/conf/templates/repository/conf/identity/charon-config.xml.j2

```diff
@@ -18,13 +18,21 @@
   -->
 
 <provisioning-config>
-    {% for key,value in identity.charon.properties.items()%}
-    <Property name="{{key}}">{{value}}</Property>
-    {% endfor %}
+    <Property name="user-schema-extension-enabled">{{charon.properties.user_schema_extension.enabled}}</Property>
+    <Property name="bulk-maxOperations">{{charon.properties.bulk_operations.max}}</Property>
+    <Property name="bulk-maxPayloadSize">{{charon.properties.bulk_payload_size.max}}</Property>
+    <Property name="patch-supported">true</Property>
+    <Property name="documentationUri">http://example.com/help/scim.html</Property>
+    <Property name="bulk-supported">true</Property>
+    <Property name="filter-supported">true</Property>
+    <Property name="filter-maxResults">200</Property>
+    <Property name="changePassword">true</Property>
+    <Property name="sort-supported">false</Property>
+    <Property name="etag-supported">false</Property>
     <authenticationSchemes>
-        {% for scheme in identity.charon.enabled_auth_schemes%}
-        <schema>
-            {% for key,value in identity.charon.authentication_schemes[scheme].items()%}
+        {% for scheme in charon.enabled_auth_schemes%}
+        <schema id="{{loop.index}}">
+            {% for key,value in charon.authentication_schemes[scheme].items()%}
```

Comment:
Seems we won't need to loop the scemas, so let's have the two known schemas only

Reply:
Fixed.